library/Mockery.php

```diff
@@ -858,7 +858,8 @@ private static function getNewDemeterMock(
                 if ($parRefMethodRetType !== null) {
                     $nameBuilder = new MockNameBuilder();
                     $nameBuilder->addPart('\\' . $newMockName);
-                    $mock = self::namedMock($nameBuilder->build(), (string) $parRefMethodRetType);
+                    $type = PHP_VERSION_ID >= 70100 ? $parRefMethodRetType->getName() : (string)$parRefMethodRetType;
```

Comment:
Maybe it's worth having a utility method in this library which does this, instead of copying the logic 4 times?

Reply:
I think it's not worth it and would be easier to remove them anytime soon when the minimum PHP version gets increased.